### ISSUE TITLE
The dimension of shape and expression space

### ISSUE THREAD

**Issue by Archer666** [TimoBolkart/TF_FLAME#2]
Thanks your code, I notice the dimension of shape space is 300, and the dimension of expression is 100, Can I change the dimension? How to do that?

**Comment by TimoBolkart** [comment]
Yes, the available FLAME model provides 300 shape components, and 100 expression components. Changing the shape or expression basis (i.e. to get more basis vectors) requires re-training the model.

**Comment by Archer666** [comment]
Thanks your reply，can you share the training code? I need to train and test on CoMA dataset. Thanks again.

**Comment by TimoBolkart** [comment]
As you see in the FLAME paper, the model is trained from more than 33 000 scans of about 4000 subjects. Especially the shape space is trained from about 4000 subjects. Also, FLAME disentangles identity, pose and expression which will not be possible from a dataset of only 12 subjects. Also, the FLAME training code is based on the SMPL training code, which we cannot release.
